src/challenges.js

```diff
@@ -1,21 +1,37 @@
 /* ======================== CallBacks Practice ============================ */
 const each = (elements, cb) => {
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
+  }
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
 };
 
 const map = (elements, cb) => {
```

Comment:
Great job.  If you come back on this try and use the `each` function you created above to still pass the `map` test.  Otherwise great job.